packages/types/lib/sync/api.ts

```diff
@@ -1,5 +1,22 @@
 import type { ApiError, Endpoint } from '../api.js';
 
+export type PostTrigger = Endpoint<{
```

Comment:
Can you also add it to `PublicApiEndpoints` in api.endpoints.ts
```suggestion
export type PostPublicTrigger = Endpoint<{
```